OptimoveSDK/optimove-sdk/src/main/res/xml/optimove_backup_rules.xml

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <full-backup-content>
-    <exclude domain="sharedpref" path="com.optimove.android.user_ids.xml"/>
-    <exclude domain="sharedpref" path="com.optimove.android.optitrack_shared_pref.xml"/>
-    <exclude domain="sharedpref" path="com.optimove.android.local_init.xml"/>
-    <exclude domain="sharedpref" path="com.optimove.android.registration_preferences.xml"/>
+    <exclude domain="sharedpref" path="com.optimove.sdk.core.xml"/>
```

Comment:
the goal is to
- exclude all SDK shared preferences from backup
- keep old keys for backwards compat
- remove unused, add not added

LGTM!